docs/learn/advanced/01-transactions.md

```diff
@@ -201,6 +201,79 @@ Each gRPC method has its corresponding REST endpoint, generated using [gRPC-gate
 
 An example can be seen [here](https://docs.cosmos.network/main/user/run-node/txs#using-rest)
 
+##### Data Conversion Flow for `/cosmos/tx/v1beta1/txs` Endpoint
+
+To handle the JSON to Protobuf conversion for the `/cosmos/tx/v1beta1/txs` endpoint. Here are the steps to follow:
+
+1. Parse the JSON to Structs:
+Continue parsing the JSON string into Go structs as previously done.
+
+```go
+jsonString := `{
+    "from_address": "cosmos1...",
+    "to_address": "cosmos1...",
+    "amount": [{"denom": "stake", "amount": "50"}],
+    "memo": "Test transaction"
+}`
+
+var jsonTx struct {
+    FromAddress string       `json:"from_address"`
+    ToAddress   string       `json:"to_address"`
+    Amount      []tx.Coin    `json:"amount"`
+    Memo        string       `json:"memo"`
+}
+json.Unmarshal([]byte(jsonString), &jsonTx)
```

Comment:
It says "as previously done", but I don't see any other example doing this. I also don't think this is correct, the tx structure doesn't look right and the type `tx.Coin` doesn't exist in the SDK (or at least we don't usually use `tx.` with it).

Later also `tx.MsgSend` is used, which should use a more familiar import path for bank types.